ansible/inventories/staging/hosts

```diff
@@ -102,11 +102,15 @@ catalogharvester2d.dev-ocsit.bsp.gsa.gov
 
 [redis:children]
 redis-catalog
+redis-catalog-next
+redis-inventory-next
 
 [redis-catalog]
 catalog-harvester1d.dev-ocsit.bsp.gsa.gov
 
 [redis-catalog-next]
+redis1d.dev-ocsit.bsp.gsa.gov
+
 [redis-inventory-next]
 
```

Comment:
Is there a reason we didn't add the appropriate box here? If not redis1d, is there a redis2d? Still in work?

Reply:
Right now there is only one redis instance. I'm waiting until we actually configure catalog-next with redis because I'm not sure if inventory needs a separate instance or not. If we can configure CKAN with specific Redis database instances, then we can use the same instance for catalog and inventory.